apps/jobs/views.py

```diff
@@ -528,8 +528,18 @@ def get_submission_by_pk(request, submission_id):
                 description='Submission error file content'
             ),
             'result': openapi.Schema(
-                type=openapi.TYPE_STRING,
-                description='Submission results'
+                type=openapi.TYPE_OBJECT,
+                description='Submission results (The keys in this string must correspond to the leaderboard schema format. There can be n number of keys.)',
+                properties={
+                    'key 1': openapi.Schema(
```

Comment:
Let's rename the `key 1` to `metric-1` and similar for the `key 2`. It will become much easier in that case for someone to understand what the format is.